pydiggy/node.py

```diff
@@ -642,25 +356,6 @@ def _explode(
                         obj[key] = value._asdict()
         return obj
 
-    @classmethod
-    def create(cls, **kwargs) -> Node:

```

Comment:
Doesn't this break the API?

```python
class MyNode(Node):
    pass

my_node = MyNode.create(foo="bar")
```

Reply:
Ah... yes, it does. Before I run off an fix it, though: Is there a reason that this logic isn't in the __init__ method? That's where I'm used to putting constructor-type logic.

Reply:
So that in the future it might also be able to handle an `awaitable` interface:

```python
my_node = await MyNode.create(foo="bar")
```

No way to achieve this with `__init__`.